<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_cadUser</class>
 <widget class="QDialog" name="form_cadUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cadastro de Usuários</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nome:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtNome"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Senha:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtSenha">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Repetir Senha:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtRepetSenha">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Tipo de usuário</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbTipo">
       <item>
        <property name="text">
         <string>admin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tecnico</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>user</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCad">
       <property name="text">
        <string>Cadastrar</string>
       </property>
       <property name="icon">
        <iconset resource="../resource/icons.qrc">
         <normaloff>:/imagens/icons/1493238513_Check.png</normaloff>:/imagens/icons/1493238513_Check.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSair">
       <property name="text">
        <string>Encerrar</string>
       </property>
       <property name="icon">
        <iconset resource="../resource/icons.qrc">
         <normaloff>:/imagens/icons/1493332785_button_cancel.png</normaloff>:/imagens/icons/1493332785_button_cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/icons.qrc"/>
 </resources>
 <connections/>
</ui>
